R/filter_irrelevant_rules.R

```diff
@@ -1,9 +1,10 @@
 #' @title
-#' Prune (Causal) Decision Rules
+#' Filter Irrelevant Decision Rules
```

Comment:
Why it is irrelevant? Is it a word that is used in the paper or literature. Pruning does make more sense in this context.

Reply:
There are several possible criteria to prune a tree. Here in particular we are pruning the irrelevant rules defined as the rules with no significant improvement for prediction (see 'Interpreting tree ensembles with inTrees' by Houtao Deng, 2019). I have already added this reference in the function description. 
I called the function filter_irrelevant also for consistency with the following filtering criteria